src/OrbitBase/File.cpp

```diff
@@ -6,6 +6,17 @@
 
 #include "OrbitBase/Logging.h"
 
+#if defined(__linux)
+#include <unistd.h>
+#elif defined(_WIN32)
+#include <io.h>
+#endif
+
+#if defined(_WIN32)
+// Windows does not have TEMP_FAILURE_RETRY - define a shortcut
```

Comment:
It's not really a shortcut, more like a fallback.

Reply:
I still have in my todo to look if EINTR is generated by windows, I suspect this error code is not at all used. Does anyone happen to know btw?

Reply:
I think you can ignore EINTR. It can't occur on Windows (Ref: https://docs.microsoft.com/en-us/cpp/c-runtime-library/errno-constants?view=msvc-160)